MapboxDirections/MBAnnotation.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  MBAnnotationType.swift
+//  MapboxDirections
+//
+//  Created by Bobby Sudekum on 3/20/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+import Foundation
+
+@objc(AnnotationType)
+public enum AnnotationType: Int, CustomStringConvertible {
+
+    case congestion
```

Comment:
"Congestion" is a strange word to use on its own, since it's uncountable. (Imagine talking about "congestions".) Let's call it `congestionLevel`.